populus/cli/deploy_cmd.py

```diff
@@ -55,119 +60,23 @@ def echo_post_deploy_message(web3, deployed_contracts):
         "mainnet' and 'morden' are pre-configured to connect to the public "
         "networks.  Other values should be predefined in your populus.ini"
     ),
+    default="tester",
```

Comment:
I don't think that this can have a default.  It doesn't make sense to me for there to be a default chain for deployments unless it's explicitly configured by the user in their config file or something.

Reply:
part of the efforts to avoid prompts